src/SearchPage.tsx

```diff
@@ -0,0 +1,37 @@
+import { Suspense, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+
+import { InputSection } from "./components/InputSection";
+import { ListSection } from "./components/ListSection/ListSection";
+
+export const SearchPage = () => {
+  const [, setSearchParams] = useSearchParams();
+
+  const [keyword, setKeyword] = useState<string>("");
```

Comment:
url의 쿼리 파라미터를 이용해 검색어 키워드를 관리하도록 설정한 것은, url을 전달하며 검색 결과도 함께 전달할 수 있도록 하기 위함이에요. 그런데 현재는 쿼리 파라미터를 검색어의 기본값으로 내려받지 못하는 문제가 발생했어요. 

TODO: 서치파라미터 값을 키워드 초기값으로 설정

```suggestion
  const [searchParams, setSearchParams] = useSearchParams();

  const [keyword, setKeyword] = useState<string>(searchParams.get("keyword") || "");
```